Description: change the default config file location
 redsocks by default looks for ./redsocks.conf. Change this to
 /etc/redsocks.conf for a more deterministic behaviour.
Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Forwared: no
Last-Update: 2013-04-23
--- a/main.c
+++ b/main.c
@@ -45,7 +45,7 @@ app_subsys *subsystems[] = {
 	&dnstc_subsys,
 };
 
-static const char *confname = "redsocks.conf";
+static const char *confname = "/etc/redsocks.conf";
 static const char *pidfile = NULL;
 
 static void terminate(int sig, short what, void *_arg)
